build/packer/filebeat/filebeat-agent.yml

```diff
@@ -24,5 +24,4 @@ processors:
 output.logstash:
   enabled: true
   timeout: 15
-  # Static Interal IP of ELK instance
-  hosts: ["10.124.0.2:5044"]
+  hosts: ["triptolemus.cockroachlabs.com:5044"]
```

Comment:
Can use [DNS peering](https://cloud.google.com/dns/docs/overview#dns-peering) to resolve internal hosts across GCP projects.